opentelemetry-etw-logs/src/logs/reentrant_logprocessor.rs

```diff
@@ -19,13 +15,8 @@ pub struct ReentrantLogProcessor {
 
 impl ReentrantLogProcessor {
     /// constructor
-    pub fn new(
-        provider_name: &str,
-        event_name: String,
-        provider_group: ProviderGroup,
-        exporter_config: ExporterConfig,
-    ) -> Self {
-        let exporter = ETWExporter::new(provider_name, event_name, provider_group, exporter_config);
+    pub fn new(provider_name: &str) -> Self {
```

Comment:
will you be fixing the implementation to invoke the actual export method? like done in user-events ?
https://github.com/open-telemetry/opentelemetry-rust-contrib/blob/main/opentelemetry-user-events-logs/src/logs/reentrant_logprocessor.rs#L27-L30

Reply:
Nothing visibly slow. In the even this turns out to be an issue, we can always model exporter as a processor itself. We don't expose anything other than `with_exporter` in public API for this reason.

https://github.com/open-telemetry/opentelemetry-rust/issues/2031 has some prior discussions.